Lib/sqlite3/__init__.py

```diff
@@ -24,18 +24,18 @@
 The sqlite3 extension module provides a DB-API 2.0 (PEP 249) compilant
 interface to the SQLite library, and requires SQLite 3.7.15 or newer.
 
-To use the module, you must first create a database Connection object:
+To use the module, start by creating a database Connection object:
 
     import sqlite3
     cx = sqlite3.connect("test.db")  # test.db will be created or opened
 
-You can also use the special database name ":memory:" to connect to a transient
-in-memory database:
+Use the special path name ":memory:" to connect to a transient in-memory
+database:
```

Comment:
Same as above -- I don't really like the passive, but I think it's clearer than the the imperative here.

```suggestion
The special path name ":memory:" can be provided to connect to a transient
in-memory database:
```